Formula/swiftgen.rb

```diff
@@ -1,7 +1,7 @@
 class Swiftgen < Formula
   desc "Swift code generator for assets, storyboards, Localizable.strings, …"
   homepage "https://github.com/SwiftGen/SwiftGen"
-  url "https://github.com/SwiftGen/SwiftGen.git", :tag => "4.2.0"
+  url "https://github.com/SwiftGen/SwiftGen.git", :tag => "4.2.1"
```

Comment:
Didn't think it was mandatory if you specify the tag, but I just added it anyway.